<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>428</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabLetras">
      <property name="tabPosition">
       <enum>QTabWidget::West</enum>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="tabNotas" native="true">
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>400</height>
       </size>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>0</y>
         <width>471</width>
         <height>429</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>3</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetNoConstraint</enum>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <property name="spacing">
           <number>3</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetNoConstraint</enum>
          </property>
          <item>
           <layout class="QFormLayout" name="formLayout">
            <property name="horizontalSpacing">
             <number>3</number>
            </property>
            <property name="verticalSpacing">
             <number>3</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Apellido</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="txtApellido"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Nombres</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="txtNombres"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="Telefono">
              <property name="text">
               <string>Telefono</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="txtTelefono"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
             <number>3</number>
            </property>
            <item>
             <widget class="QTextEdit" name="txtDatosPersonales">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>50</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnGrabar">
              <property name="text">
               <string>Guardar</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="spacing">
           <number>3</number>
          </property>
          <item>
           <layout class="QFormLayout" name="formLayout_3">
            <property name="horizontalSpacing">
             <number>3</number>
            </property>
            <property name="verticalSpacing">
             <number>3</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="Mes">
              <property name="text">
               <string>Mes</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <property name="spacing">
               <number>3</number>
              </property>
              <item>
               <widget class="QComboBox" name="cboMes">
                <item>
                 <property name="text">
                  <string>Enero</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Febrero</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Marzo</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Abril</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Mayo</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Junio</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Julio</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Agosto</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Septiembre</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Octubre</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Noviembre</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Diciembre</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btnPagar">
                <property name="text">
                 <string>Pagar</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout_2">
            <property name="horizontalSpacing">
             <number>3</number>
            </property>
            <property name="verticalSpacing">
             <number>3</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Nota</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <property name="spacing">
               <number>3</number>
              </property>
              <item>
               <widget class="QLineEdit" name="txtNuevaNota_2"/>
              </item>
              <item>
               <widget class="QPushButton" name="btnAddNota_2">
                <property name="text">
                 <string>Agregar</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextEdit" name="txtNotas">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>300</height>
           </size>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Archivo">
    <property name="title">
     <string>&amp;Archivo</string>
    </property>
    <addaction name="action_Salir"/>
   </widget>
   <widget class="QMenu" name="menu_Personas">
    <property name="title">
     <string>&amp;Personas</string>
    </property>
    <addaction name="action_Nueva"/>
    <addaction name="action_Borrar"/>
   </widget>
   <widget class="QMenu" name="menuListados">
    <property name="title">
     <string>Listados</string>
    </property>
    <addaction name="actionPagos"/>
   </widget>
   <addaction name="menu_Archivo"/>
   <addaction name="menu_Personas"/>
   <addaction name="menuListados"/>
  </widget>
  <action name="action_Guardar">
   <property name="text">
    <string>&amp;Guardar</string>
   </property>
  </action>
  <action name="action_Salir">
   <property name="text">
    <string>&amp;Salir</string>
   </property>
  </action>
  <action name="action_Nueva">
   <property name="text">
    <string>&amp;Nueva</string>
   </property>
  </action>
  <action name="action_Borrar">
   <property name="text">
    <string>&amp;Borrar</string>
   </property>
  </action>
  <action name="action_Guardar_2">
   <property name="text">
    <string>&amp;Guardar</string>
   </property>
  </action>
  <action name="actionPagos">
   <property name="text">
    <string>Pagos...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
